layers/sflow.go

```diff
@@ -1696,24 +1696,24 @@ func decodeExtendedUserFlow(data *[]byte) (SFlowExtendedUserFlow, error) {
 //  Packet IP version 4 Record
 // **************************************************
 
-//  0                      15                      31
-//  +--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+
-//  |                     Length                    |
-//  +--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+
-//  |                    Protocol                   |
-//  +--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+
-//  |                  Source IPv4                  |
-//  +--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+
-//  |                Destination IPv4               |
-//  +--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+
-//  |                   Source Port                 |
-//  +--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+
-//  |                Destionation Port              |
-//  +--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+
-//  |                   TCP Flags                   |
-//  +--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+
-//  |                      TOS                      |
-//  +--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+
+// 0                      15                      31
```

Comment:
I think the things in this file should be indented to be `monospace`

Reply:
how do you mean? looks like it's gone from spaces to tabs which is a go recommended way?

Reply:
Oh, I think this was a different error than what I was thinking (and one not intruduced in this PR).  I was thinking these were godoc comments, like this:

```
//<one space>SFlowIpv4Record foos the bar
//<two spaces>packet format
```

If it had been, then that would have made the packet format `monospace` in the godoc.  However, the lack of the first line with a `//<single space>godoc start` invalidates that.  As is, all of these will (continue to) render in poorly-formatted non-monospace and probably look really weird in the godoc. (confirmed:  https://pkg.go.dev/github.com/google/gopacket@v1.1.19/layers#SFlowIpv4Record)

If you'd like to leave as is, tots fine.  However, if you're in here and want things to look a little nicer, you could change

```
//<two spaces>packet format
```

to

```
//<one space><Struct Name>
//
//<tab>packet format
```

Reply:
aaaah I got what you mean now. Happy to include it in the current PR